packages/contracts/package.json

```diff
@@ -2,12 +2,12 @@
   "name": "contracts",
   "packageManager": "yarn@4.1.0",
   "scripts": {
-    "test": "forge test -vv",
-    "test:watch": "forge test -vv --watch",
+    "test": "forge test -vvv",
```

Comment:
Lucas, what made the change of -vv to -vvv necessary?

Reply:
to log the errors, see [docs](https://book.getfoundry.sh/forge/tests#logs-and-traces)